### PR TITLE
[fix bug 1342235] Redesigned site footer

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new universal site footer to all pages for `en` locales, falling back to the regular footer for other locales.
- Site footer is now also theme-able, undoing many years over style overrides 😉 

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1342235

## Testing
Things worth testing:
- en-US footer responsiveness [pebbles]
- non-en footer responsiveness [pebbles]
- en-US footer responsiveness [sandstone]
- non-en footer responsiveness [sandstone]
- RTL footer responsiveness [sandstone]
- oldIE footer [sandstone]

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by chrismore** [comment]
@craigcook @alexgibson What's the status of this review? This was supposed to be reviewed, merged and released by the end of this week. Thanks

**Comment by craigcook** [comment]
> @craigcook @alexgibson What's the status of this review? This was supposed to be reviewed, merged and released by the end of this week. Thanks

Slight delay while I was swamped with other priorities, but I'm reviewing today and we should be able to merge and push to production first thing Monday.

File: media/css/pebbles/components/_footer.scss [hunk 1]
```diff
@@ -43,6 +46,174 @@
...
+            margin-bottom: 0;
+        }
+    }
+
+    &.dark,
+    &.gray,
+    &.blue {
+        @media #{$mq-tablet} {
+            .primary {
+                border-bottom: 1px solid #fff;
```
Comment (craigcook) [inline media/css/pebbles/components/_footer.scss]: Totally a nitpick but it looks like the Sandstone version doesn't have this rule? The Firefox pages with a blue footer have a gray line that doesn't look as nice as the white.

**Comment by alexgibson** [comment]
@craigcook added some review fixes - thanks

**Pull request merged**